content/applications/productivity/iot/devices/screen.rst

```diff
@@ -2,34 +2,36 @@
 Connect a screen
 ================
 
-In Odoo, an :abbr:`IoT (Internet of Things)` Box can be connected to a screen display. After being
-connected, the screen can be used to display a :abbr:`PoS (Point of Sale)` order to a client.
+In Odoo, an :abbr:`IoT (Internet of Things)` box can be connected to a screen display. After being
+configured, the screen can be used to display a :abbr:`PoS (Point of Sale)` order to a client.
 
 .. image:: screen/screen-pos-client-display.png
    :align: center
    :alt: An example of a PoS (point of sale) order on a screen display.
 
 .. note::
    Access the customer display from any other computer by going to the :abbr:`IoT (Internet of
-   Things)` Box homepage and clicking on the :guilabel:`POS Display` button.
+   Things)` box homepage and clicking on the :guilabel:`POS Display` button. Access the :abbr:`IoT
+   (Internet of Things)` box homepage by navigating to :menuselection:`IoT app --> IoT Boxes` and
+   click on the :abbr:`IP (Internet Protocol)` or :abbr:`URL (Uniform Resource Locators)`.
 
 Connection
 ==========
 
-The way to connect the screen display to the :abbr:`IoT (Internet of Things)` Box differs depending
+The way to connect the screen display to the :abbr:`IoT (Internet of Things)` box differs depending
 on the model.
 
 .. tabs::
 
    .. tab:: IoT Box model 4
 
       Connect up to two screens with Micro-HDMI cables on the side of the :abbr:`IoT (Internet of
-      Things)` Box. If two screens are connected, they can display distinct content (see usage
+      Things)` box. If two screens are connected, they can display distinct content (see usage
       below).
 
    .. tab:: IoT Box model 3
 
-      Connect the screen with an HDMI cable on the side of the :abbr:`IoT (Internet of Things)` Box.
+      Connect the screen with an HDMI cable on the side of the :abbr:`IoT (Internet of Things)` box.
 
 .. important::
    Screen(s) should be connected before the :abbr:`IoT (Internet of Things)` Box is switched on. If
```

Comment:
Decapitalizing "box". There's another instance of this in this "Important" box that needs to be corrected as well.
```suggestion
   Screen(s) should be connected before the :abbr:`IoT (Internet of Things)` box is switched on. If
```